lenovo/thinkpad/l14-amd/default.nix

```diff
@@ -0,0 +1,24 @@
+{ config, lib, pkgs, ... }:
+
+{
+  imports = [
+    ../.
+    ../../../common/cpu/amd
```

Comment:
Could their be a generic l14 module that gets extended by the more specific amd profile?

Reply:
So I've created `l14/amd` and `l14/intel`. I've populated the `l14/intel` profile with the same content as the `l13` profile, because its basically the same laptop in a smaller chassis, _but_ I couldn't test it.